package.json

```diff
@@ -59,6 +59,12 @@
           "scope": "resource",
           "description": "List of local or remote style sheets for suggestions.",
           "default": []
+        },
+        "vscode-html-css.vaildOnSave": {
+          "type": "boolean",
+          "description": "Verify label class names when saving files.",
+          "scope": "application",
```

Comment:
Better to be resource (workspace) scopped and be an enum for future. (Like On save, On change etc)